README.md

```diff
@@ -66,6 +66,9 @@ hubname = "azure-hub-name"
 resourcegroup = "azure-resource-group"
 mqttport = 8883
 
+# Set the server hostname
+# hostname = "myhostname.local"
```

Comment:
I intentionally left this disabled since I didn't want to force people into an arbitrary choice. It's there just to let people know the option exists, but if we're going to default to anything, the system `hostname` respond (option 3 in the order options are evaluated) probably makes the most sense, and `localhost` as the worst case fallback.